consensus/src/transactions.rs

```diff
@@ -1,94 +1,104 @@
+//! # Transaction Verifier Service.
+//!
+//! This module contains the [`TxVerifierService`] which handles consensus validation of transactions.
+//!
 use std::{
     collections::HashSet,
     future::Future,
     ops::Deref,
     pin::Pin,
-    sync::Arc,
+    sync::{Arc, Mutex as StdMutex},
```

Comment:
Why `Mutex` -> `StdMutex`?

Reply:
to make it clear it's not an async mutex